src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php

```diff
@@ -69,6 +83,15 @@ public function get_item_permissions_check( $request ) {
 		return $this->check_has_read_only_access( $request );
 	}
 
+	public function check_permission_menu( $menu_id ) {
+		$nav_term = wp_get_nav_menu_object( $menu_id );
+		if ( $nav_term && isset( $nav_term->show_in_rest ) && $nav_term->show_in_rest ) {
+			return true;
+		}
+
+		return false;
```

Comment:
Nitpick: I wonder if this could be refactored to:
```suggestion
		return $nav_term && isset( $nav_term->show_in_rest ) && $nav_term->show_in_rest;
```

Reply:
Honestly, I prefer things like this in the conditional rather as part of the `return`.  This makes it more clear what is being returned.